<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>ReplaceDialog</class>
 <widget class="QDialog" name="ReplaceDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>452</width>
    <height>277</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Replace Text</string>
  </property>
  <widget class="QWidget" name="horizontalLayout_2" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>10</y>
     <width>401</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayout_3" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>50</y>
     <width>401</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_3" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>90</y>
     <width>431</width>
     <height>131</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" >
    <property name="margin" >
     <number>0</number>
    </property>
    <property name="spacing" >
     <number>6</number>
    </property>
    <item>
     <widget class="QGroupBox" name="groupBox" >
      <property name="title" >
       <string>Options</string>
      </property>
      <widget class="QWidget" name="layoutWidget_4" >
       <property name="geometry" >
        <rect>
         <x>10</x>
         <y>30</y>
         <width>191</width>
         <height>77</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="QCheckBox" name="checkWords" >
          <property name="text" >
           <string>Whole words only</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkCase" >
          <property name="text" >
           <string>Case sensitive</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBegin" >
          <property name="text" >
           <string>Start at Beginning</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_2" >
      <property name="title" >
       <string>Direction</string>
      </property>
      <widget class="QWidget" name="layoutWidget_5" >
       <property name="geometry" >
        <rect>
         <x>10</x>
         <y>30</y>
         <width>191</width>
         <height>52</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="QRadioButton" name="radioForward" >
          <property name="text" >
           <string>Forward</string>
          </property>
          <property name="checked" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioBackward" >
          <property name="text" >
           <string>Backward</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_2" >
   <property name="geometry" >
    <rect>
     <x>11</x>
     <y>11</y>
     <width>431</width>
     <height>71</height>
    </rect>
   </property>
   <layout class="QGridLayout" >
    <property name="margin" >
     <number>0</number>
    </property>
    <property name="spacing" >
     <number>6</number>
    </property>
    <item row="0" column="1" >
     <widget class="QComboBox" name="comboFind" >
      <property name="sizePolicy" >
       <sizepolicy>
        <hsizetype>7</hsizetype>
        <vsizetype>0</vsizetype>
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="editable" >
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="1" >
     <widget class="QComboBox" name="comboReplace" >
      <property name="sizePolicy" >
       <sizepolicy>
        <hsizetype>7</hsizetype>
        <vsizetype>0</vsizetype>
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="editable" >
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="0" >
     <widget class="QLabel" name="label_2" >
      <property name="sizePolicy" >
       <sizepolicy>
        <hsizetype>1</hsizetype>
        <vsizetype>1</vsizetype>
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text" >
       <string>Replace</string>
      </property>
      <property name="buddy" >
       <cstring>comboReplace</cstring>
      </property>
     </widget>
    </item>
    <item row="0" column="0" >
     <widget class="QLabel" name="label" >
      <property name="sizePolicy" >
       <sizepolicy>
        <hsizetype>1</hsizetype>
        <vsizetype>1</vsizetype>
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text" >
       <string>Find</string>
      </property>
      <property name="buddy" >
       <cstring>comboFind</cstring>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>230</y>
     <width>431</width>
     <height>33</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" >
    <property name="margin" >
     <number>0</number>
    </property>
    <property name="spacing" >
     <number>6</number>
    </property>
    <item>
     <spacer>
      <property name="orientation" >
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" >
       <size>
        <width>131</width>
        <height>31</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="findButton" >
      <property name="text" >
       <string>Find</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="replaceallButton" >
      <property name="text" >
       <string>Replace All</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="closeButton" >
      <property name="text" >
       <string>Close</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <pixmapfunction></pixmapfunction>
 <tabstops>
  <tabstop>comboFind</tabstop>
  <tabstop>comboReplace</tabstop>
  <tabstop>checkWords</tabstop>
  <tabstop>checkCase</tabstop>
  <tabstop>checkBegin</tabstop>
  <tabstop>radioForward</tabstop>
  <tabstop>radioBackward</tabstop>
  <tabstop>findButton</tabstop>
  <tabstop>replaceallButton</tabstop>
  <tabstop>closeButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
